db-service/package.json

```diff
@@ -22,7 +22,7 @@
     "CHANGELOG.md"
   ],
   "scripts": {
-    "test": "jest --silent"
+    "test": "cds-test --skip toMatchSnapshot"
```

Comment:
please also create a follow up. Otherwise these tests are only run with our internal pipeline.

Reply:
I personally don't see any value in the static snapshot tests. The problem of that databases is not their ability to render `SQL` it will always be determined by the integration with the database.

Here are some examples why the current state of the static `cqn2sql` tests don't provide any value.

1. test found a gap in the `CREATE` cqn got skipped instead of fixed ([code](https://github.com/cap-js/cds-dbs/blob/d25af70b23688cd22c7b87f20a7848c9653ae9a9/db-service/test/cqn2sql/create.test.js#L45))
1. tests the `not in` operator, but never found that it does not work on HANA ([code](https://github.com/cap-js/cds-dbs/blob/d25af70b23688cd22c7b87f20a7848c9653ae9a9/db-service/test/cqn2sql/delete.test.js#L57))
1. Uses legacy `cqn` format in many expression `['(',...,')']` ([code](https://github.com/cap-js/cds-dbs/blob/d25af70b23688cd22c7b87f20a7848c9653ae9a9/db-service/test/cqn2sql/delete.test.js#L80))

So while there is value in the `cqn` that the tests define. They are not actually being tested to determine functional correctness. Therefor the most beneficial action here would be to go over all the `cqn2sql` tests and move them into their respective `compliance` test suites.